javase/src/test/java/net/glxn/qrgen/javase/QRCodeTest.java

```diff
@@ -245,6 +247,36 @@ public void shouldGetSvgWithSizeAndColorFromText() {
         assertThat(file.length()).isGreaterThan(0);
     }
 
+    @Test
+    public void shouldGetSvgFromTextAsStream() {
+        ByteArrayOutputStream outs = new ByteArrayOutputStream();
+        QRCode.from("www.example.org").svg(outs);
+        byte[] data = outs.toByteArray();
+        assertThat(data).isNotEmpty();
+        assertThat(data.length).isGreaterThan(0);
```

Comment:
Doesn't above condition isNotEmpty handle it?

Reply:
yeah, removed